### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by jdvala** [facebookresearch/MUSE#50]
I am trying to create my own bilingual word embeddings and I am finding it difficult to understand somethings. I will try and describe my problem in detail in hope to find some answers
I have a parallel corpus and I  want to create bilingual word embeddings, I preprocessed both of them and created my word vectors with gensim(I wish I have used fasttext, but same again the docs were not clear) Now I have both word embeddings, the question I have is if I dont have same number of words in both the models, does this work for MUSE?

**Comment by glample** [comment]
Yes you can have an arbitrary number of words for each model this is not a problem at all.
